xml/System.Runtime.InteropServices/UnmanagedType.xml

```diff
@@ -605,7 +605,7 @@
       </ReturnValue>
       <MemberValue>47</MemberValue>
       <Docs>
-        <summary>A Windows Runtime string. You can use this member on the <see cref="T:System.String" /> data type.</summary>
+        <summary>A Windows Runtime string. You can use this member on the <see cref="T:System.String" /> data type. [Built-in support for WinRT is removed from .NET 5](/dotnet/core/compatibility/interop/5.0/built-in-support-for-winrt-removed).</summary>
```

Comment:
I think the link above is appropriate. Can you also add a link to the workaround in our best-practices guide?

https://learn.microsoft.com/dotnet/standard/native-interop/best-practices#formerly-built-in-supported-types